contrib/platform/src/com/sun/jna/platform/mac/CoreFoundation.java

```diff
@@ -488,6 +488,9 @@ public static CFStringRef createCFString(String s) {
          */
         public String stringValue() {
             CFIndex length = INSTANCE.CFStringGetLength(this);
+            if (length.longValue() == 0) {
```

Comment:
Debating whether this can/should be combined with the maxSize check below, but I think I like it the way you've done it here.

Reply:
It could reduce code, but this function is very used in my cenario, I prefer evict another native call.
I discovered this bug when extract the title of another programs windows with macOS accessibility framework.